var/spack/repos/builtin/packages/sw4lite/package.py

```diff
@@ -52,6 +52,14 @@ def build_targets(self):
             cxxflags.append('-DSW4_CROUTINES')
             targets.append('ckernel=yes')
 
+        if '+cuda' in self.spec:
+            targets.append('NVCC = {0}'.format(
+                self.spec['cuda'].prefix.bin.nvcc))
+            targets.append('HOSTCOMP = {0}'.format(self.compiler.cxx))
```

Comment:
@ax3l @svenevs should this be `self.compiler.cxx` (the actual compiler) or `spack_cxx` (the compiler wrapper)?

Reply:
Hmm. I'm not really sure, I guess it feels more spack-like to make both the wrappers? I don't know of any other examples that need to do this though, I think everything else just relies on things being (injected) in the PATH?